### PR TITLE
Fix var types of parameters in `sanitize_option()` and `sanitize_option_{$option}` filter

### PR DESCRIPTION
**Pull request by crstauf**
Fixes the documentation of the parameters used in `sanitize_option_{$option}` filter.

Reported by `gerardreches` via User Contributed Notes.

Trac ticket: https://core.trac.wordpress.org/ticket/60214

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/formatting.php [hunk 1]
```diff
@@ -5119,9 +5119,9 @@ function sanitize_option( $option, $value ) {
 	 * @since 2.3.0
 	 * @since 4.3.0 Added the `$original_value` parameter.
 	 *
-	 * @param string $value          The sanitized option value.
-	 * @param string $option         The option name.
-	 * @param string $original_value The original value passed to the function.
+	 * @param mixed $value          The sanitized option value.
+	 * @param string $option        The option name.
+	 * @param mixed $original_value The original value passed to the function.
```
Comment (mukeshpanchal27) [inline src/wp-includes/formatting.php:5124]: The function `sanitize_option` docblock said that it's string do we needs to update that also? 🤔

**Comment by mukeshpanchal27** [comment]
@crstauf left one minor question for clarification. Thanks!

File: src/wp-includes/formatting.php (hunk 1 above)
Comment (crstauf) [reply src/wp-includes/formatting.php:5124]: Great idea; done.

**Comment by crstauf** [comment]
> left one minor question for clarification. Thanks!

@mukeshpanchal27 Done. Thanks.

**Comment by audrasjb** [comment]
Committed in https://core.trac.wordpress.org/changeset/57302

**Comment by sheebaawp** [comment]
same issue facing again

{{{
#!html
<a href="https://tripmegamart.africa/"></a>
<a href="https://tripmegamart.ae/"></a>
<a href="https://tripmegamart.us/"></a>
<a href="https://tripmegamart.eu/"></a>
<a href="https://tripmegamart.com/"></a>
<a href="https://watansky.com/"></a>
<a href="https://banglalyriczone.com/"></a>
<a href="https://aitamil.com"></a>
<a href="https://biologyranker.com"></a>
}}}

**Pull request closed without merging**